app/views/govuk_publishing_components/components/docs/label.yml

```diff
@@ -65,3 +65,12 @@ examples:
       text: "Label with an ID"
       id: "id-for-the-label"
       html_for: "id-that-matches-input-6"
+  with_dir_attribute:
+    description: |
+      To allow the correct display of right to left languages.
```

Comment:
Maybe add something like `Dir also applies to label hint, as shown.`?

Reply:
Yes, good point. Have added that. 👍